vello/src/lib.rs

```diff
@@ -452,6 +434,15 @@ impl Renderer {
     /// The texture is assumed to be of the specified dimensions and have been created with
     /// the [`wgpu::TextureFormat::Rgba8Unorm`] format and the [`wgpu::TextureUsages::STORAGE_BINDING`]
     /// flag set.
+    ///
+    /// If you want to render Vello content to a surface (such as in a UI toolkit), you have two options:
+    /// 1) Render to an intermediate texture, which is the same size as the surface.
+    ///    You would then use [`TextureBlitter`][wgpu::util::TextureBlitter] to blit the rendered result from
+    ///    that texture to the surface.
+    ///    This pattern is supported by the [`util`] module.
+    /// 2) Call `render_to_texture` directly on the [`SurfaceTexture`][wgpu::SurfaceTexture]'s texture, if
+    ///    it has the right usages. This should generally be avoided, as it's a potential performance pitfall
+    ///    on GPUs where the render pipeline method of writing to surfaces is assumed and optimised for.
```

Comment:
Something about this phrasing is confusing to me.

Reply:
I've tried to rephrase this.